src/main/java/Legin.java

```diff
@@ -0,0 +1,232 @@
+import exception.LeginMissingParamsException;
+import exception.LeginEmptyTaskException;
+import exception.LeginIndexOutOfBoundsException;
+import exception.LeginInvalidCommandException;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.io.FileNotFoundException;
+import java.io.File;
+import java.io.FileWriter;
+import java.util.Scanner;
+
+public class Legin {
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+    private static int currentTaskCount = 0;
+    private static final String FILE_PATH = "./tasklist.txt";
+
+    public static void horizontalLine() {
```

Comment:
You would want to use verbs to name a method instead of nouns.
```suggestion
    public static void printHorizontalLine() {
```